File: cpp/2780.cpp
//2780	Basquete de Robôs	Accepted	C++	0.000	19/10/2018 16:46:04
#include <iostream>

using namespace std;

int main()
{
    int n;
    
    cin >> n;
    
    if(n<=800)
        cout<<"1\n";
    else if(n>800 && n<=1400)
        cout<<"2\n";
    else
        cout<<"3\n";

    return 0;
}

File: Go/2780.go
//2780	Basquete de Robôs	Accepted	Go	0.208	19/10/2018 16:39:14
package main

import (
	"fmt"
)

func main() {

	var a int
	
	fmt.Scanf("%d", &a)

	if a<=800{
		fmt.Printf("1\n")
	}else if a>800 && a<=1400{
		fmt.Printf("2\n")
	}else{
		fmt.Printf("3\n")
	}
}